pkg/container-utils/testutils/options.go

```diff
@@ -60,12 +62,24 @@ func WithContext(ctx context.Context) Option {
 	}
 }
 
+func WithExpectStartError() Option {
+	return func(opts *containerOptions) {
+		opts.expectStartError = true
+	}
+}
+
 func WithImage(image string) Option {
 	return func(opts *containerOptions) {
 		opts.image = image
 	}
 }
 
+func WithBindMounts(mounts []string) Option {
```

Comment:
nit: Perhaps of passing a string we can define a new struct that contains source and target fields?

Reply:
I don't know if it is worth it.
There is also a `Mount struct`:
https://github.com/moby/moby/blob/v26.1.4/api/types/mount/mount.go#L24-L31
But it is Docker specific and this `containerOptions struct` is used for both containerd and docker.